kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/filtered-resources/kjars-sources/cloudbalance/src/main/resources/META-INF/kmodule.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<kmodule xmlns="http://jboss.org/kie/6.0.0/kmodule">
```

Comment:
As of 6.4.0.Beta1, the kmodule xmlns namespace has switched to "http://www.drools.org/xsd/kmodule"
We should never use "http://jboss.org/kie/6.0.0/kmodule" again (except in the regression tests in drools itself because it still works for backwards compatibility)